### PR TITLE
[Mux] remove unnecessary var

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Code clean, remove unnecessary data in mux collect-pad data.

### CODE REVIEW

**Review by anyj0527 — APPROVED** [review]
LGTM👍

**Review by niley7464 — APPROVED** [review]
LGTM!

File: gst/nnstreamer/nnstreamer_plugin_api_impl.c [hunk 1]
```diff
@@ -389,8 +389,8 @@ gst_tensor_time_sync_buffer_from_collectpad (GstCollectPads * collect,
     data = (GstCollectData *) walk->data;
     pad = (GstTensorCollectPadData *) data;
```
Comment (songgot) [inline gst/nnstreamer/nnstreamer_plugin_api_impl.c:390]: I think this line should be deleted too. Am I right? :)

Comment (jaeyun-jung) [reply gst/nnstreamer/nnstreamer_plugin_api_impl.c:390]: no, this tensorpaddata is used to handle sync mode now. (it has old gst-buffer for time sync mode)

Comment (songgot) [reply gst/nnstreamer/nnstreamer_plugin_api_impl.c:390]: Oh, I see, I misread the {} range. Thanks for checking. :smiley:

**Review by songgot — APPROVED** [review]
LGTM :satisfied:

**Pull request merged**